resources/views/page.blade.php

```diff
@@ -125,6 +125,10 @@
             @endif
         </div>
         <!-- /.content-wrapper -->
+        
+        <footer class="main-footer">
```

Comment:
After adding this, I get a footer but it is blank and I dont know where to edit my footer

Reply:
You can edit the footer global in the page.blade.php or you can overwrite the footer on every page.